GOVERNANCE.md

```diff
@@ -78,7 +84,8 @@ including:
 * Conduct guidelines
 * Maintaining the list of additional Collaborators
 
-* [Current list of TSC members](./README.md#current-project-team-members)
+The current list of TSC members can be found
+[here](./README.md#current-project-team-members).
```

Comment:
Generally, it's good to avoid "here" as link text. Maybe use "in the project README" or something like that instead?